### PR TITLE
Hotfix/ens registrar

### PR DESCRIPTION
**Pull request by gamalielhere**
### Hotfix

- [x] Updated CHANGELOG.md  
- [x] Add PR label

### CODE REVIEW

File: src/dapps/RegisterDomain/RegisterDomain.vue [hunk 1]
```diff
@@ -145,12 +154,28 @@ export default {
           break;
         case '4':
           this.loading = false;
+          this.highestBidder = unit.fromWei(res[4], 'ether').toString();
```
Comment (kvhnuke) [inline src/dapps/RegisterDomain/RegisterDomain.vue]: `this.highestBid` might be a better name

File: src/dapps/RegisterDomain/containers/EnsBidContainer/EnsBidContainer.vue [hunk 2]
```diff
@@ -322,17 +324,19 @@ export default {
         data: this.raw['data'],
         from: this.raw['from'],
         to: this.raw['to'],
-        value: this.raw['value'],
+        value: !this.$route.fullPath.includes('reveal') ? this.raw['value'] : 0,
```
Comment (kvhnuke) [inline src/dapps/RegisterDomain/containers/EnsBidContainer/EnsBidContainer.vue]: I really dont like this way of checking `path.includes` shouldn't the value be 0 if it is reveal anyway?

File: src/dapps/RegisterDomain/containers/EnsBidContainer/EnsBidContainer.vue [hunk 3]
```diff
@@ -322,17 +324,19 @@ export default {
...
         gasPrice: this.raw['gasPrice'],
         gas: this.raw['gas'],
         nonce: this.raw['nonce']
       };
-      const doc = new jsPDF();
-      doc.fromHTML(this.$refs.printableData, 15, 15, {
-        width: 300
-      });
+      if (!this.$route.fullPath.includes('reveal')) {
```
Comment (kvhnuke) [inline src/dapps/RegisterDomain/containers/EnsBidContainer/EnsBidContainer.vue]: same here

**Comment by gamalielhere** [comment]
Includes #140, #122, #129, #126

**Pull request merged**